### DISCUSSION TITLE
GRASSHOPPER CATEGORIES

### DISCUSSION THREAD

**Discussion by JBENEITEZ** [ENZYME-APD/archicad-api-dev#46] [General]
@grzegorzwilk @thekaushikls @leceta I was thinking these days that we could slightly rearrange the categories for the nodes. Currently:
- Attribute
- Data Input
- Classification
- Properties
- selection
- BIMx

My proposal is to increase the number of categories: 
- Project & File Information
- Libraries & Teamwork
- Selection and filtering
- Properties and Classification
- Attributes
 -Views and Layouts
- Viewpoints
- Geometry: General
- Geometry: 3D By type
- Geometry: 2D
- GDL and objects
- Interoperability (IFC & others)
- QA&QC (Quality Control)
- Schedules and QTO
- BIMx

**Comment by thekaushikls** [comment]
Hi @JBENEITEZ Although I like the idea of grouping by functionality, feel it is a bit too early to write in concrete about the exact number of components, and to which category / sub-category they might fall into. I'd suggest that we push this to the second milestone, since this involves re-structuring the components. What do you think?

**Reply by JBENEITEZ** [reply]
Makes sense!
